src/main/java/org/testobject/rest/api/resource/v2/ApiTestReportResource.java

```diff
@@ -15,13 +15,13 @@ public ApiTestReportResource(WebTarget target) {
 		this.target = target;
 	}
 
-	public TestReportWithDevice getTestReport(long reportId, String apiKey) {
+	public AppiumTestReport getTestReport(long reportId, String apiKey) {
```

Comment:
is this always an AppiumTestReport?

Reply:
Yes. Weird, isn't it? As if the `testobject-java-api` was a bit more attuned only to Appium testing.